recipe/meta.yaml

```
@@ -50,13 +51,112 @@ requirements:
     - libxml2
     - draco
     - openssl
-  run_constrained:   # [osx and x86_64]
-    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+    - pdal
+
+  outputs:
```

This should be at the first indent.

```suggestion
outputs:
```